core/src/main/java/co/aikar/commands/annotation/Syntax.java

```diff
@@ -26,6 +26,11 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * Specifies the syntax to be used when executing this command.
+ * It should not include any descriptions of the arguments nor when some are allowed and when they are not.
```

Comment:
Use of this annotation is not necessary. Syntax will be automatically generated for you.
Use this annotation to override automatic syntax